### PR TITLE
Remove `Telegraf::handleUpdates`

### PR DESCRIPTION
**Pull request by wojpawlik**
One closure and one private method less.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/telegraf.ts [hunk 1]
```diff
@@ -92,11 +91,11 @@ export class Telegraf<C extends Context = Context> extends Composer<C> {
     return this.telegram.token
   }
 
-  /** @deprecated use `ctx.telegram.webhookReply` */
   set webhookReply(webhookReply: boolean) {
     this.telegram.webhookReply = webhookReply
   }
 
+  /** @deprecated use `ctx.telegram.webhookReply` */
```
Comment (EdJoPaTo) [inline src/telegraf.ts:143]: Changing these deprecation hints looks unrelated to the PR? (same with L95)
Also I'm not sure why one (set or get) is deprecated but not both?

Comment (wojpawlik) [reply src/telegraf.ts:143]: Thanks for reviews!

Yes, it's unrelated. I just fix small issues in the files I'm already editing.

Getter has to be deprecated, otherwise someone can think that the property is becoming read-only. Setter does not, write-only properties make very little sense, but might as well.

**Review by MKRhere — APPROVED** [review]
